frontend/javascripts/admin/api/jobs.ts

```diff
@@ -179,12 +179,37 @@ export function startNeuronInferralJob(
   layerName: string,
   bbox: Vector6,
   newDatasetName: string,
+  doSplitMergerEvaluation: boolean,
+  annotationId?: string,
+  useSparseTracing?: boolean,
+  evalMaxEdgeLength?: string,
+  evalSparseTubeThresholdNm?: string,
+  evalMinMergerPathLengthNm?: string,
```

Comment:
The typing is incorrect here: the parameters are actually parsed as numbers here. Please see my other comments regarding improving the typing
```suggestion
  evalMaxEdgeLength?: number,
  evalSparseTubeThresholdNm?: number,
  evalMinMergerPathLengthNm?: number,
```